superset/tasks/cache.py

```diff
@@ -316,8 +316,11 @@ def cache_warmup(strategy_name, *args, **kwargs):
     return results
 
 
-@celery_app.task(name='datasources-cache')
-def update_datasources_cache(cache_timeout=24 * 60 * 60):
+def update_datasources(cache_timeout=24 * 60 * 60):
```

Comment:
Rather than having `cli` take a dependency on `tasks`, consider moving this into a shared library that they both depend on so that these two things aren't unnecessarily coupled. (Maybe something under `utils` if there is nothing better in the current code structure?)

Reply:
Good point, I'll move it to a third location.